zproject/settings.py

```diff
@@ -1302,6 +1302,6 @@ def get_secret(key):
 # This is a debugging option only
 PROFILE_ALL_REQUESTS = False
 
-CROSS_REALM_BOT_EMAILS = set(('feedback@zulip.com', 'notification-bot@zulip.com'))
+CROSS_REALM_BOT_EMAILS = set(('feedback@zulip.com', 'notification-bot@zulip.com', 'welcome-bot@zulip.com'))
```

Comment:
Yeah, I think adding a similar exception for PMs makes sense, rather than adding it as a cross-realm bot. Though I'm not too familiar with this part of the system, so might be worth asking in #development help.